pkg/client/k8s/k8s.go

```diff
@@ -71,28 +71,33 @@ const (
 	PersistentVolumeKK K8sKind = "PersistentVolume"
 	// PersistentVolumeClaimKK is a K8s PersistentVolumeClaim Kind
 	PersistentVolumeClaimKK K8sKind = "PersistentVolumeClaim"
-	// CstorPoolCRKK is a K8s CR of kind CStorPool
+	// CStorPoolCRKK is a K8s CR of kind CStorPool
 	CStorPoolCRKK K8sKind = "CStorPool"
 	// DiskCRKK is a K8s CR of kind Disk
 	DiskCRKK K8sKind = "Disk"
-	// CstorVolumeCRKK is a K8s CR of kind CStorVolume
+	// CStorVolumeCRKK is a K8s CR of kind CStorVolume
 	CStorVolumeCRKK K8sKind = "CStorVolume"
-	// CstorVolumeReplicaCRKK is a K8s CR of kind CStorVolumeReplica
+	// CStorVolumeReplicaCRKK is a K8s CR of kind CStorVolumeReplica
 	CStorVolumeReplicaCRKK K8sKind = "CStorVolumeReplica"
 )
 
-//
+// K8sAPIVersion is the type declaration for API version information
 type K8sAPIVersion string
 
 const (
+	// ExtensionsV1Beta1KA is the extensions/v1beta` API
```

Comment:
remove backtick after v1beta1

Reply:
just pushed a second commit to fix this, thanks.